### ISSUE TITLE
Problem debuging EXI locally

### ISSUE THREAD

**Issue by igvaz** [ispyb/EXI#530]
Hello! 

I'm trying to debug EXI to run with ISPYB but I'm getting these errors when "npm install" it 

![image](https://user-images.githubusercontent.com/61015285/226664498-601950a1-ab39-4681-b3d4-ba030d6d8988.png)

Don't know how to proceed with that :( can anybody help please?

**Comment by antolinos** [comment]
Hi,

There are many deprecation warnings but I do not see the error. What is the error?
Once said this, EXI is not maintained and we strongly recommend to use the new UI:
https://github.com/ispyb/py-ispyb-ui

**Comment by igvaz** [comment]
> Hi,
> 
> There are many deprecation warnings but I do not see the error. What is the error? Once said this, EXI is not maintained and we strongly recommend to use the new UI: https://github.com/ispyb/py-ispyb-ui

Hello @antolinos! 

Understand, we're trying to debug ISpyB so we can use it here in LNLS - SIRIUS. 

I fixed the problem with the deprecated warnings, now got this one: 
![image](https://user-images.githubusercontent.com/61015285/226676296-242b0283-a2ad-4b97-bb8b-4f3f079d4fba.png)

But since you recommend to use the py-ispyb-ui, I'll try to do it 

Another doubt: Is py-ispyb backend fully functional and may I use it? Or keep with ISpyB + py-ispyb-ui? 

Thanks for your attention in advance

**Comment by antolinos** [comment]
Exact, ISPyB with py-ispyb-ui if you want to use for MX (that I think it is the case).
Honestly, I'd not spend time on EXI.
